Documentation updates and fixes

AratKruglik opened the pull request.

AratKruglik committed:
chore: Add documentation for frontend architecture and testing strategy

AratKruglik committed:
chore: Update contributing and security guidelines

AratKruglik committed:
Merge branch 'develop' into feature/project-documentation

kennwat commented on docs/SECURITY_GUIDELINES.md:
```diff
@@ -143,7 +143,7 @@ class NoScriptTag implements Rule
     'min:12',
     'confirmed',
-    'regex:/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]/',
+    'lax',
```
тут замість "lax" мала б бути регулярка, яка валідує пароль:
'^regex:/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{12,}$/'
а в розділі Session Security:
'same_site' => 'strict' поміняти на 'same_site' => 'lax',

AratKruglik replied:
Дякую! Апдейтнув.

AratKruglik committed:
chore: Update security guidelines for password validation and session cookie settings

AratKruglik committed:
Merge branch 'develop' into feature/project-documentation

AratKruglik committed:
docs: Docs update, AI agents guidelines update

AratKruglik committed:
Merge remote-tracking branch 'origin/feature/project-documentation' into feature/project-documentation

# Conflicts:
#	.claude/agents/code-reviewer.md
#	.cursor/mcp.json
#	.junie/guidelines.md
#	.junie/mcp/mcp.json
#	.vscode/mcp.json
#	README.md
#	docs/CONTRIBUTING.md

AratKruglik committed:
chore: Rename `compose.yml` to `docker-compose.yml` and update relevant documentation

AratKruglik committed:
chore: Remove outdated development guidelines and MCP configuration files

AratKruglik committed:
chore: Update MCP configuration to use docker-based commands and add new server definitions

maks-oleksyuk commented on README.md:
```diff
@@ -5,305 +5,113 @@
-потужні інструменти для розробки через Model Context Protocol.
+- **PHP 8.4+**
+- **Composer**
+- **Node.js** з Yarn 4.6.0
```
або вже вказати версію node та yarn нову або видалти їх взагалі

AratKruglik replied:
Виправив

maks-oleksyuk commented on docker-compose.yml:
якщо я не помиляюсь це тепер офіційний неймінг для такого типу файлів, а це виглядає як крок назад

AratKruglik replied:
Виправив

maks-oleksyuk commented on CLAUDE.md:
```diff
@@ -276,6 +276,57 @@ php artisan view:clear
+- laravel/mcp (MCP) - v0
+- laravel/pint (PINT) - v1
+- laravel/telescope (TELESCOPE) - v5
+- pestphp/pest (PEST) - v3
```
v4

AratKruglik replied:
Виправив

maks-oleksyuk commented on CLAUDE.md:
```diff
@@ -276,6 +276,57 @@ php artisan view:clear
+- laravel/pint (PINT) - v1
+- laravel/telescope (TELESCOPE) - v5
+- pestphp/pest (PEST) - v3
+- phpunit/phpunit (PHPUNIT) - v11
```
v12 починаючи з pest v4

AratKruglik replied:
Вирпавив

maks-oleksyuk commented on CLAUDE.md:
```diff
@@ -276,6 +276,57 @@ php artisan view:clear
+## Foundational Context
+This application is a Laravel application and its main Laravel ecosystems package & versions are below. You are an expert with them all. Ensure you abide by these specific packages & versions.
+
+- php - 8.4.13
```
чи є сенс вказувати патч версію?

AratKruglik replied:
Виправив

AratKruglik committed:
Merge branch 'develop' into feature/project-documentation

AratKruglik committed:
Merge branch 'refs/heads/develop' into feature/project-documentation

# Conflicts:
#	composer.lock

AratKruglik committed:
docs: update project documentation and add Docker Compose configuration

AratKruglik committed:
Merge remote-tracking branch 'origin/feature/project-documentation' into feature/project-documentation

AratKruglik committed:
chore: Update configuration file path in XDEBUG documentation

AratKruglik committed:
chore: Rename docker-compose.yml to compose.yml in docs

Updated references from 'docker-compose.yml' to 'compose.yml' in migration documentation.

AratKruglik committed:
chore: remove unused jobs and related tests from the codebase

maks-oleksyuk commented on api.http:
шо то за файл і нащо він треба?

maks-oleksyuk commented on README.md:
````diff
@@ -5,305 +5,129 @@
-docker compose exec app ./vendor/bin/pest --mutate --covered-only --parallel --min=100
-```
-
-## DDEV Інсталяція
````
а нащо це видалили? це корисне
